civic_europe/assets/js/bootstrap5-fix.js

```diff
@@ -1,29 +1,37 @@
+// This file runs on every load and checks if there are any bootstrap 5
+// data attributes on any DOM elements. It checks for data-bs-toggle,
+// data-bs-dismiss and data-bs-target attributes and converts them to
+// bootstrap 4 equivalent attributes.
+
 const bs5ToggleElements =
```

Comment:
@phillimorland at the end I decided to keep them separated (e.g. running it three times), but using the methods getAttribute, setAttribute and removeAttribute, which is much better!

But reducing it to one method bloats the code, because then one has to find out the actual attribute `toggle`, `dismiss` and `target` and assign the right values to them `modal`, `dropdown` and `#map-import-modal`.